src/test/init/features/project.spec.ts

```diff
@@ -53,8 +56,8 @@ describe("project", () => {
   describe("getProjectInfo", () => {
     it("should get correct project info when no project supplied", async () => {
       const options = {};
-      listProjectsStub.returns([TEST_FIREBASE_PROJECT, ANOTHER_FIREBASE_PROJECT]);
-      promptStub.returns("my-project-123");
+      listProjectsStub.resolves([TEST_FIREBASE_PROJECT, ANOTHER_FIREBASE_PROJECT]);
```

Comment:
Some of these were changed to `resolves` and others kept as `returns` (like in above file at `resolvePpathStub.returns`) - why were some changed and others not? Is this an API change necessitated by the version bump or is there more to this?

Reply:
so `returns` is actually incorrect since the methods should return a `Promise` (which is what `resolves` does. The ones that broke the TS compiler were the ones I fixed.

Reply:
Weird, it seemed to have no compiling problems before? Did we update typescript? I'm very cautious about sinon and other js/ts testing tools because from my experience it's super easy to use completely the wrong method and never know it as tests will keep passing - an example is asserting something when you're not actually calling the assert method.